stock_getter/backend_stockgetter/test_config.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import configparser
+
+# Test if local configuration file meet the requirementsa and ready to use in app 
+config = configparser.ConfigParser()
+config.read('api_keys.ini')
```

Comment:
Just to check, this is a test for someone to make sure their created file is good? We no longer have this file anywhere in the repo.

Reply:
Yep, this is only for users to test if their file is good